Webex Connect Flows/v3.0/Template/Event Handling Workflows/README.md

```diff
@@ -0,0 +1,155 @@
+# Event Handling Workflows
+
+## Workflows Overview
+These flows are optional unless flow admin needs to build custom logic like displaying a Screen pop on the Agent's desktop etc. on occurrence of certain events. With v3.0 flows there will be default handling for all the events like
+adding participant , removing participant , closing the conversation etc. If flow admin needs any alternative handling or additional business logic then these flows can be imported and customized as per the requirement.
+These event handling flows are media channel agnostic and if different handling is required for channels or assets, then one can deploy as many flows as per the use case.
+
+### Routed Event flow
+* This workflow will be triggered when an agent is successfully assigned to a contact. The event for this workflow is 'Task Routed'. So once the contact has been routed to the agent and the
+agent has clicked on the accept button then this workflow will be triggered.
+
+#### When do you need Routed Event Flow ?
+
+* Routed event flow is an optional flow .It is needed only when you want to do some customization like screen pop or http calls once the agent has clicked on the accept button.
```

Comment:
Routed event flow is an optional flow . -> Routed event flow is an optional flow.